src/Mvc/test/Mvc.FunctionalTests/HtmlGenerationTest.cs

```diff
@@ -118,26 +114,34 @@ public async Task HtmlGenerationWebSite_GeneratesExpectedResults(string action,
             responseContent = responseContent.Trim();
             if (antiforgeryPath == null)
             {
-#if GENERATE_BASELINES
                 ResourceFile.UpdateFile(_resourcesAssembly, outputFile, expectedContent, responseContent);
-#else
                 Assert.Equal(expectedContent.Trim(), responseContent, ignoreLineEndingDifferences: true);
-#endif
             }
             else
             {
                 var forgeryToken = AntiforgeryTestHelper.RetrieveAntiforgeryToken(responseContent, antiforgeryPath);
-#if GENERATE_BASELINES
-                // Reverse usual substitution and insert a format item into the new file content.
-                responseContent = responseContent.Replace(forgeryToken, "{0}");
-                ResourceFile.UpdateFile(_resourcesAssembly, outputFile, expectedContent, responseContent);
-#else
-                expectedContent = string.Format(CultureInfo.InvariantCulture, expectedContent, forgeryToken);
-                Assert.Equal(expectedContent.Trim(), responseContent, ignoreLineEndingDifferences: true);
-#endif
+
+                if (ResourceFile.GenerateBaselines)
+                {
+                    // Reverse usual substitution and insert a format item into the new file content.
+                    responseContent = responseContent.Replace(forgeryToken, "{0}");
+                    ResourceFile.UpdateFile(_resourcesAssembly, outputFile, expectedContent, responseContent);
+                }
+                else
+                {
+                    expectedContent = string.Format(CultureInfo.InvariantCulture, expectedContent, forgeryToken);
+                    Assert.Equal(expectedContent.Trim(), responseContent, ignoreLineEndingDifferences: true);
+                }
             }
         }
 
+        [ConditionalTheory]
+        [InlineData("Link", null)]
+        [InlineData("Script", null)]
+        [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/10423")]
```

Comment:
Not going to block, but does this work now that the content root is correct? Just in case that caused these tests to weird out.

Reply:
Ah good point, I will try that in the next iteration, I need to try running these on the arm queues too anyways before merging, since I vaguely recall functional tests like this not working at all on arm queues due to some publishing quirks

Reply:
Looks like the content root can't have a trailing '/' as the FileProviders throw with that appended.  Is there a different place where we can add the / so the file resolution stuff deals with it properly?